test/routers/confirmCompany.int.ts

```diff
@@ -14,6 +14,10 @@ mockGetCompanyProfile.mockResolvedValue(validCompanyProfile);
 
 describe("ConfirmCompany router/handler integration tests", () => {
 
+    beforeAll(() => {
+        // middlewareMocks.mockCsrfProtectionMiddleware.mockClear();
```

Comment:
comment can be removed, and by extension, the `beforeAll` block (if there's no code in it)

Reply:
Thanks, i'll fix this